test/integration/ControllerBalancerV2Strat-test.js

```diff
@@ -0,0 +1,121 @@
+const {
+  createController,
+  createPiToken,
+  deploy,
+  getBlock,
+  waitFor,
+  zeroAddress
+} = require('../helpers')
+
+const { resetHardhat, setCustomBalanceFor, setChainlinkRoundForNow } = require('./helpers')
+
+describe('Controller BalancerV2 Strat', () => {
+  let bob
+  let piToken
+  let archimedes
+  let controller
+  let strat
+  let rewardsBlock
+  let USDC
+  let QI
+  let BAL
+  let USDCFeed
```

Comment:
Since they are no `const` I will put them in lowercase, even if they "deserve" uppercase =)